index.js

```diff
@@ -276,7 +325,7 @@ var getTzDistFilename = function (tzid) {
 var getDataSource = function (source) {
   var geoJson
   if (source.source === 'overpass') {
-    geoJson = require('./downloads/' + source.id + '.json')
+    geoJson = require(argv.downloads_dir + '/' + source.id + '.json')
```

Comment:
If I try putting in the path `downloads-test` without the `./` prefix this will fail. If this is changed to `path.resolve(argv.downloads_dir)`, then it will work. It might be a good idea to wrap each of these arguments that are folders with a path.resolve to help with this.

Reply:
I've done as you suggested.